pages/common/ugrep.md

```diff
@@ -0,0 +1,36 @@
+# ugrep
+
+> Ultra fast search tool with query TUI.
+> More information: <https://github.com/Genivia/ugrep>.
+
+- Start a query TUI to search files in the current directory recursively (press Control-Z for help):
+
+`ugrep --query`
+
+- Search the current directory recursively for files containing a regex search pattern:
+
+`ugrep "{{search_pattern}}"`
+
+- Search a specific file or search all files in the specified directory, showing line numbers of matches:
+
+`ugrep --line-number "{{search_pattern}}" {{path/to/file_or_directory}}`
+
+- List the matching files in the current directory recursively:
+
+`ugrep --files-with-matches "{{search_pattern}}"`
+
+- Fuzzy search files with up to 3 extra, missing or mismatching characters in the pattern:
+
+`ugrep --fuzzy=3 "{{search_pattern}}"`
+
+- Search compressed files, zip and tar archives recursively:
+
+`ugrep --decompress "{{search_pattern}}"`
+
+- Search only files whose filenames match the `foo*.???` glob pattern:
```

Comment:
```suggestion
- Search for filenames which match a glob pattern:
```

Reply:
My original wording here is the same as the GNU grep man page wording, which is accurate, because this option applies a filter to only search files whose filenames match the glob, where the pattern (not the glob) is the search pattern to search for files (but not we only search those whose filenames match the glob). The change suggests that we now search for files that match the given glob pattern, but that is something different.

Reply:
That's right, but I think we should still keep the example a little more general:
```suggestion
- Search only files whose filenames match a specific glob pattern:
```

Reply:
Yes, I agree. The `foo*.???` was suggested by another reviewer so I replaced `glob_pattern` with that.